### ISSUE TITLE
Buttons and links should inform a user what they are

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1364]
NEED SOMEONE RESPONSIBLE FOR WRITING GOOD COPY HERE

All text for buttons or links should explain what they are without additional context

**Acceptance Criteria:**
- "Find out more" on the homepage, maybe should be "Learn about us!" or something
- "Get started" on support your team could be "Start designing your curriculum!"
- "Sign up" on aboutus/workwithus could be "Become a subject expert"
- "Sign up" on the newsletter component could be "Sign up to newsletter"

**Comment by jack-skerrett-oak** [comment]
https://www.notion.so/oaknationalacademy/The-label-used-for-the-Find-out-more-button-is-too-generic-and-doesn-t-specify-to-the-user-what-th-7d7ac04354dd4b508417c46f893ceacc?pvs=4
https://www.notion.so/oaknationalacademy/The-label-used-for-the-Get-Started-button-is-too-generic-and-doesn-t-specify-to-the-user-what-they-f588a1367c994e199fee93fa793242a8?pvs=4
https://www.notion.so/oaknationalacademy/The-label-used-for-the-Sign-up-button-is-too-generic-and-doesn-t-specify-to-the-user-what-they-wil-1c724df51d4e479db6938a233ccca273?pvs=4
https://www.notion.so/oaknationalacademy/The-Sign-up-button-text-is-too-generic-and-doesn-t-specify-to-users-what-they-are-signing-up-for-3a2bab1fbe614d099cf50d863b40c1d4?pvs=4

**Comment by jack-skerrett-oak** [comment]
moved to notion
